lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -137,6 +137,22 @@ def initialize(model, current_user:, embed_links: false)
           }
         end
 
+        link :'custom-fields' do
+          {
+            href: 'custom-fields',
+            type: 'text/html',
+            title: "Custom fields"
```

Comment:
@oliverguenther Are you sure? Wouldn't it have to show "Custom fields" just like that (without any I18n)? However, it shows an translation error.